.editorconfig

```diff
@@ -0,0 +1,24 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+
+[**.{md,markdown}]
+trim_trailing_whitespace = false
+
+[{lib,src,test}/**.js]
+indent_style = space
+indent_size = 2
+
+[src/{**.h,**.cc}]
+indent_style = space
+indent_size = 2
+
+[test/*.py]
+indent_style = space
+indent_size = 4
+
+[Makefile]
```

Comment:
If you end up moving indent_size = 2 to *, can you add 8 here? (Ok, call me a unix beard but it's at least relatively default in make-land)